src/jsifier.js

```diff
@@ -422,6 +422,8 @@ function JSify(data, functionsOnly) {
     // "Final shape that will be created").
     print('// EMSCRIPTEN_END_FUNCS\n');
 
+    if (ASSERTIONS) print('var ASSERTIONS = true;\n');
+    print(read('arrayUtils.js'));
```

Comment:
The style looks like some sort of preprocessor syntax, but it's actually just valid JS:

<img width="364" alt="screen shot 2017-06-29 at 1 43 20 pm" src="https://user-images.githubusercontent.com/415435/27702098-fbe6e724-5cd0-11e7-95aa-04cd2b3ebf52.png">